### ISSUE TITLE
[BUG] VoiceOver on Range fails to read out all values

### ISSUE THREAD

**Issue by chi-jyskebank-dk** [kirbydesign/designsystem#3774]
- [x] I have written a [good issue](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Issue) 

## Describe the bug
When operating the Range slider with VoiceOver enabled, not all values of the slider are read out loud

## Describe how to reproduce the bug

1. Go to this StackBlitz: https://stackblitz.com/edit/stackblitz-starters-wxb9gweu?file=src%2Fmain.ts
2. Operate slider with VoiceOver enabled
3. Notice, that values > 5000 and < 105000 are not read out loud

## Which Kirby version was used?
10.2.1

## What was the expected behavior?
All values the Range slider can have should be read out loud

## Additional context
The Range where the error was detected has a minimum of 0, a maximum of 500000 and a step-value of 5000.

## Add any screenshots
See attached screen recording

## Please complete the following information:
- OS: macOS 15.2 with VoiceOver
- Browser: Google Chrome

## Are there any additional context?

<hr />

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#the-process-of-contributing) correctly.

### Verification
To make sure the bug is not intended behaviour; it should be verified by a member of team Kirby before moving on to implementation. 

- [ ] Make sure the `NOT verified` label has been removed by a member of [team Kirby](https://github.com/kirbydesign/designsystem/blob/develop/.github/SUPPORT.md#team-kirby); do not proceed until this is done.

### Implementation 
The contributor who wants to implement this issue should: 

- [ ] Make sure you have read: "[Before you get coding](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#before-you-get-coding)".
- [ ] Signal to others you are working on the issue by assigning yourself.
- [ ] Create a branch from the [develop branch](https://github.com/kirbydesign/designsystem/tree/develop) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [ ] Create a test that reproduces the bug following guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)". 
- [ ] Write code to fix the bug

### Review
Once the issue has been implemented and is ready for review:

- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [ ] Create a pull-request.

https://github.com/user-attachments/assets/cd9551e9-81cc-4336-9001-30b43a945393

**Comment by RasmusKjeldgaard** [comment]
Also verify this behavior on NVDA, according to the WAS a similar issues should exist in some of the solutions that use Kirby.

**Comment by Loujuu** [comment]
The bug regarding not all values of the slider are read out loud, can not be reproduced. All values are read out loud, so this issue will be closed with no bug.

**Comment by RasmusKjeldgaard** [comment]
This is still an issue, but turns out it only happens with OS language and VO speech set to danish.
The problem also exists for `<input type="range>` so it seems likely that this is a VoiceOver bug.

I can reproduce with VO on MacOS 26.0.1 and with Safari or Chrome (v141) on this simple example with a builtin range: https://codepen.io/rasmooose/pen/dPGKaqZ

**Comment by RasmusKjeldgaard** [comment]
I have reported to Apple on https://feedbackassistant.apple.com/ but unfortunately issues are private. Will update here if there is any progress.
